environment_cuda.yml

```
@@ -0,0 +1,15 @@
+name: mtenn
+channels:
+  - pytorch
+  - pyg
+  - conda-forge
+  - nvidia
+dependencies:
+  - pytorch-cuda
```

I think this needs to have `pytorch` and `pytorch-cuda`
```suggestion
  - pytorch
  - pytorch-cuda
```